root/etc/e-smith/events/actions/nethserver-directory-tlspolicy

```diff
@@ -22,7 +22,18 @@
 
 tlspolicy=$(/sbin/e-smith/config getprop tls policy)
 
-if [[ -n ${tlspolicy} ]] && (( $tlspolicy >= 20180621 )); then
+if [[ -n ${tlspolicy} ]] && (( $tlspolicy >= 20200510 )); then
```

Comment:
Provide a default value if `tlspolicy` is an empty string:

```suggestion
if (( ${tlspolicy:-0} >= 20200510 )); then
```

Reply:
I vote for Stephane implementation: is much more simple to read!

Reply:
I vote my review: it is much more shorter and beautiful :hugs:

Reply:
Something new to learn I vote for your review

Reply:
done